### PR TITLE
fix minor typo

### LINKED ISSUE #328: Minor doc issue: service_instance vs. service_instances field
There are several terminology mistakes in the 2.13 schema.  For example, the [schema object](https://github.com/openservicebrokerapi/servicebroker/blob/v2.13/spec.md#schema-object) documentation makes reference to a `service_instance` field.  The [documentation for that field is titled  **Service Instances Object**](https://github.com/openservicebrokerapi/servicebroker/blob/v2.13/spec.md#service-instances-object) (note the pluralization).

This appears to be true for service bindings too.

Thanks for the new spec release!

### PR DESCRIPTION
**Pull request by mattmcneeney**
Fixes #328

### CODE REVIEW

**Comment by mattmcneeney** [comment]
FYI @ljnelson

**Comment by angarg12** [comment]
LGTM

**Comment by duglin** [comment]
errors look real:
```
tools/../spec.md: Can't find section '#service-instances-object' in tools/../spec.md
tools/../spec.md: Can't find section '#service-bindings-object' in tools/../spec.md
```

**Comment by mattmcneeney** [comment]
@duglin yep they were. fixed

**Comment by angarg12** [comment]
LGTM

**Comment by duglin** [comment]
LGTM merci!

**Comment by duglin** [comment]
2 reviews needed

File: spec.md [hunk 1]
```diff
@@ -333,19 +333,19 @@ how platforms might expose these values to their users.
 
 | Response field | Type | Description |
 | --- | --- | --- |
-| [service_instance](#service-instances-object) | object | The schema definitions for creating and updating a service instance. |
-| [service_binding](#service-bindings-object) | object | The schema definition for creating a service binding. Used only if the service plan is bindable. |
+| [service_instance](#service-instance-object) | object | The schema definitions for creating and updating a service instance. |
```
Comment (ljnelson) [inline spec.md]: Should be "The schema _definition_ for creating…", yes?

**Comment by vaikas-google** [comment]
LGTM

**Comment by avade** [comment]
LGTM

**Pull request merged**